src/application/Queries/Queries.tsx

```diff
@@ -1,22 +1,135 @@
-import React from "react";
-// import { useQuery } from "@apollo/client";
+/** @jsx jsx */
+import { useState } from "react";
+import { jsx, css } from "@emotion/core";
+import { useTheme } from "emotion-theming";
+import { rem } from "polished";
+import { gql, useQuery } from "@apollo/client";
+import { List } from "@apollo/space-kit/List";
+import { ListItem } from "@apollo/space-kit/ListItem";
 import { IconRun } from "@apollo/space-kit/icons/IconRun";
+import { colors } from "@apollo/space-kit/colors";
 import { SidebarLayout } from "../Layouts/SidebarLayout";
+import { QueryViewer } from "./QueryViewer";
+
+const sidebarHeadingStyles = css`
+  margin-left: ${rem(12)};
+  text-transform: uppercase;
+  font-size: ${rem(13)};
+  font-weight: normal;
+  letter-spacing: ${rem(1)};
+  color: rgba(255, 255, 255, .3);
+`;
+
+const h1Styles = css`
+  font-family: monospace;
+  font-weight: normal;
+`;
+
+const operationNameStyles = css`
+  margin: ${rem(3)} 0 0 ${rem(8)};
+  font-family: "Source Sans Pro", sans-serif;
+  color: ${colors.grey.light};
+  text-transform: uppercase;
+  font-size: ${rem(13)};
+`;
+
+const runButtonStyles = css`
+  appearance: none;
+  display: flex;
+  align-items: center;
+  margin: 0 0 0 auto;
+  border: none;
+  font-size: ${rem(15)};
+  background-color: transparent;
+  cursor: pointer;
+
+  > svg {
+    width: ${rem(16)};
+    margin-right: ${rem(8)};
+  }
+`;
+
+const listStyles = css`
+  font-family: monospace;
+  color: ${colors.silver.lighter};
+
+  > div {
+    height: ${rem(36)};
+    font-size: ${rem(15)};
+  }
+`;
+
+const GET_WATCHED_QUERIES = gql`
+  query GetWatchedQueries {
+    watchedQueries @client {
+      queries {
+        id
+        name
+      }
+    }
+  }
+`;
+
+const GET_WATCHED_QUERY = gql`
+  query GetWatchedQuery($id: ID!) {
+    watchedQuery(id: $id) @client {
+      id
+      name
+      queryString
+      variables
+      cachedData
+    }
+  }
+`;
 
 export const Queries = ({ navigationProps }) => {
+  const [selected, setSelected] = useState<number>(0);
+  const theme = useTheme<any>();
```

Comment:
To get rid of this `any`, you could narrow the types used in the `theme.ts` file a bit, then replace `any` with something that will point to one of the theme objects. In other words, you could update `src/application/theme.ts` to use `ShadedColor` as the second `Record` value type:

```ts
import { lighten } from "polished";
import { colors, ShadedColor } from "@apollo/space-kit/colors";

export enum ColorTheme {
  Light = 'light',
  Dark = 'dark'
}

const shared = {
  sidebarSelected: colors.indigo.base,
};

export const themes: Record<ColorTheme, Record<string, ShadedColor>> = {
  [ColorTheme.Light]: {
    primary: colors.indigo.darkest,
    sidebarHover: lighten(0.2, colors.indigo.darkest) as ShadedColor,
    ...shared,
  },
  [ColorTheme.Dark]: {
    primary: colors.black.base,
    sidebarHover: lighten(0.2, colors.black.base) as ShadedColor,
    ...shared,
  },
};
```

then in `Queries.tsx` reference the theme object with something like:

```ts
import { themes } from "../theme";
...
type Theme = (typeof themes)[keyof typeof themes];
```

then use this new `Theme` type like:

```ts
export const Queries = ({ navigationProps }) => {
  ...
  const theme = useTheme<Theme>();
```